tests/server/test_diff.py

```diff
@@ -70,38 +70,41 @@ async def test_list_attr_diff(client, environment, env_with_versions):
     constant_value = {"key2": "val2"}
     await create_resource_in_multiple_versions(
         env_id,
-        "/tmp/dir1",
+        "dir1",
```

Comment:
The naming should be updated to reflect the change of type i.e. `dir` and `file` is no longer accurate

Reply:
we don't really care do we? NullRescource represents nothing, so the name doensn't represent anything neither

Reply:
I like the file and dir, it gives a bit more meaning than just a,b,c